src/main/kotlin/com/expedia/graphql/extensions/GraphQLSchema.kt

```diff
@@ -0,0 +1,24 @@
+package com.expedia.graphql.extensions
+
+import graphql.schema.GraphQLSchema
+import graphql.schema.idl.SchemaPrinter
+
+private val schemaPrinter = SchemaPrinter(
+    SchemaPrinter.Options.defaultOptions()
+        .includeScalarTypes(true)
+        .includeExtendedScalarTypes(true)
+        .includeIntrospectionTypes(true)
+        .includeSchemaDefintion(true)
+        .includeDirectives(false)
+)
+
+fun GraphQLSchema.print(): String {
```

Comment:
Do we want to add the config options or accept a optional `SchemaPrinter` as input to `print()`?

Reply:
sure, can expose the boolean flags